plugins/svn/README.md

```diff
@@ -0,0 +1,10 @@
+## svn
+
+**Maintainer:** 
+
+This plugin adds some useful functions.
+
+### Usage
+
+See the [wiki](https://github.com/robbyrussell/oh-my-zsh/wiki/Plugin:svn) for a list of functions provided by the plugin.
```

Comment:
I'd rather have them listed here. We're trying to stop using the wiki for plugin documentation.